rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -923,6 +925,47 @@ protected int getReleaseRequestQueueSize() throws IOException {
         return getReleaseRequestQueue().getEntries().size();
     }
 
+    public boolean hasBridgeBtcTxSigHash(Sha256Hash sigHash) {
+        if (!activations.isActive(RSKIP379) || sigHash == null){
+            return false;
+        }
+
+        byte[] data = repository.getStorageBytes(
+            contractAddress,
+            getStorageKeyForBridgeBtcTxSigHash(sigHash)
+        );
+
+        return data != null &&
+           data.length == 1 &&
+           data[0] == TRUE_VALUE;
```

Comment:
Can We Have Tests For This?
When Data Is Null, When Data Length Is More Than 1, When Data[0] Is FALSE_VALUE; 0

Reply:
Done